decidim-proposals/app/presenters/decidim/proposals/proposal_presenter.rb

```diff
@@ -12,10 +12,13 @@ class ProposalPresenter < SimpleDelegator
       def author
         @author ||= if official?
                       Decidim::Proposals::OfficialAuthorPresenter.new
-                    elsif user_group
-                      Decidim::UserGroupPresenter.new(user_group)
                     else
-                      Decidim::UserPresenter.new(super)
+                      coauthorship = coauthorships.first
+                      if coauthorship.user_group
+                        Decidim::UserGroupPresenter.new(user_group)
```

Comment:
`coauthorship.user_group`?

Reply:
Sorry, I don't get your comment..

Reply:
Coauthorable gives access to couauthorships but also direct access to authors and to user_groups.

Reply:
You are totally right @mrcasals . Sorry, I was totally misunderstanding what you meant. It should be `coauthorship.user_group`. Fixing it right now!